.gitignore

```diff
@@ -158,3 +158,6 @@ data
 MNIST
 runs
 *trace*
+
+# neptune local files
+.neptune/
```

Comment:
Move this above to the section in L140 (pl tests)

Reply:
Done